dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -670,6 +670,13 @@
 			status = "disabled";
 			#address-cells = <1>;
 			#size-cells = <0>;
+
```

Comment:
@dleach02 a closing `};` got removed here, causing a paren balance error breaking the parse.

Reply:
Yes, Missing a }; here, I delete it carelessly when i rebase my branch, i will update this quickly.

Reply:
now I'm confused why it doesn't fail for me locally???

Reply:
Wait, nvm, I see what I did... I'm stupid with git and didn't actually checkout his branch. Sigh.